components/nimbus/tests/common/mod.rs

```diff
@@ -1,13 +1,32 @@
+#![cfg(feature = "rkv-safe-mode")]
```

Comment:
These were sort of all over in the `tests` folder so I moved them to be file annotations rather than specific lines.